src/classes/User.js

```diff
@@ -1,4 +1,3 @@
-//const allData = require("../scripts")
 const ingredientsData = require("../data/ingredients")
 const recipeData = require("../data/recipes")
```

Comment:
I cloned the branch down and it doesn't look like we're using these? I don't think we're supposed to be using any testing data (or any data that doesn't originate from the server) in any implementation code for the final project, so would it be okay to get rid of this?

Reply:
This is done. Those old imports are deleted from `User.js` and all tests are still passing.